### ISSUE TITLE
Monitoring: kube-controller-manager metrics target down - rke2ControllerManager https on port 10257 instead of 10252

### ISSUE THREAD

**Issue by websi96** [rancher/rancher#39429]
Hi,

I came across an issue with the default values of the rancher-monitoring helm chart.
In my rke2 cluster the kube-controller-manager prometheus target was down after an upgrade to kubernetes v1.22.6, because the pod was serving the metrics via https on port 10257 instead of the default port 10252 and http.

My solution was to change the [rancher-pushproxy](https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-pushprox/100.0.2) clients settings of the rke2ControllerManager on [line 218 in the values of the monitoring chart](https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-monitoring/100.1.1%2Bup19.0.3/values.yaml#L218) to use https and also changed the metricsPort to 10257  like here:

```yaml
rke2ControllerManager:
  enabled: true
  metricsPort: 10257
  component: kube-controller-manager
  clients:
    https:
      enabled: true
      insecureSkipVerify: true
      useServiceAccountCredentials: true
    port: 10011
    useLocalhost: true
    nodeSelector:
      node-role.kubernetes.io/master: "true"
    tolerations:
      - effect: "NoExecute"
        operator: "Exists"
      - effect: "NoSchedule"
        operator: "Exists"
```

I am still missing the ca certificate used for tls. I didn't know wich ca file to use from the rke2 deployment and none of the default seemed to work, so I used "insecureSkipVerify: true".

Any clues where I can find the correct ca for the kube-controller-manager metrics https endpoint?

Thanks!

**Comment by aiyengar2** [comment]
Should have been fixed starting at 100.1.1+up19.0.3 in https://github.com/rancher/charts/pull/1766

**Comment by websi96** [comment]
Awesome! Thanks :)
